### ISSUE TITLE
Jest test suite failed to run AngularJS v1.x is not loaded! after upgrade from angular 7 to 9 

### ISSUE THREAD

**Issue by gnbabu** [ui-router/angular-hybrid#457]
After upgrading @uirouter/angular-hybrid to 10. 0.0
I am getting the below error when I run npm test. we have used jest for unit test coverage.

```
Test suite failed to run AngularJS v1.x is not loaded!
  at Object.noNg (../../../packages/upgrade/src/common/src/angular1.ts:228:9)
  at node_modules/@uirouter/angular-hybrid/bundles/ng:/@uirouter/angular-hybrid/angular-hybrid.ts:51:38
  at node_modules/@uirouter/angular-hybrid/bundles/uirouter-angular-hybrid.umd.js:2:68
  at Object.<anonymous> (node_modules/@uirouter/angular-hybrid/bundles/uirouter-angular-hybrid.umd.js:5:2)
  at Object.<anonymous> (src/customer-forms/customer-forms.routing.module.ts:2:1)
  at Object.<anonymous> (src/customer-forms/index.ts:6:1)
  at Object.<anonymous> (src/core/effects/library.effects.ts:25:1)
  at Object.<anonymous> (src/core/effects/index.ts:1:1)
  at Object.<anonymous> (src/core/index.ts:41:1)
  at Object.<anonymous> (src/selectors/print-queue.selectors.ts:3:1)
  at Object.<anonymous> (src/selectors/index.ts:7:1)
  at Object.<anonymous> (src/shared/directives/order-by.directive.ts:13:1)
  at Object.<anonymous> (src/shared/directives/index.ts:2:1)
  at Object.<anonymous> (src/shared/shared.module.ts:4:1)
  at Object.<anonymous> (src/shared/index.ts:1:1)
  at Object.<anonymous> (test/effects/common.effects.test.ts:4:1)
```
It has both angulajs and angular9 upgrade code base

Jest configuration

```
"@uirouter/rx": "0.6.5", 
"@uirouter/angularjs": "1.0.26",
"@uirouter/angular": "7.0.0",
"@uirouter/core": "6.0.5",
"@uirouter/angular-hybrid": "10.0.0",
"@types/jest": "24.0.6",
"bamboo-jest-reporter": "1.0.3",
"jest": "24.9.0",
"jest-junit": "10.0.0",
"ts-jest": "24.1.0"
```

The above versions are listed in package.json. it is working in angularjs+angular7 solution after upgrade to 8 ,9 or 10 versions i am getting the above mentioned issue.
Any one has idea on this please help.

**Comment by gnbabu** [comment]
i have updated the @uirouter/angular-hybrid configuration used in the issue

**Comment by christopherthielen** [comment]
I think this is an ngUpgrade problem.  angular-hybrid calls `angular.module` when it gets loaded, but `angular` isn't loaded yet in your tests.

Try this workaround:
https://github.com/angular/angular/issues/16484

**Comment by gnbabu** [comment]
I have already using setAngularJSGlobal to register angular in main.ts

```
import angular from 'angular';
setAngularJSGlobal(angular);
```

**Comment by gnbabu** [comment]
@christopherthielen any other suggestions you can give about this issue.

**Comment by christopherthielen** [comment]
@gnbabu make sure you load `angularjs` before anything else.  This message is coming from `angular/upgrade/static` and indicates that angularjs is being used before it is loaded.  It's being used by `uirouter-hybrid` when it gets loaded. 

That's about all I can gather from your problem description.

Perhaps this article will help explain it better: https://blog.nrwl.io/ngupgrade-in-depth-436a52298a00

**Comment by gnbabu** [comment]
Issue got fixed
This issue is coming because of the paths of the services imports. We have imported service directly from the paths like bellow 

`import{ WorkPermitsService } from '../../work-permits/work-permits.service';`

After upgrading to angular 9 imported paths are not resolved when I run the unit tests. So i have exported it in services index.ts  and importing from @services (common place for exporting) where ever required. like this i have done for all.

Change :
`export { WorkPermitsService } from '../../work-permits/work-permits.service';`

Importing in required places
`import { WorkPermitsService,} from '@services';`
